apps/extension/package.json

```diff
@@ -5,9 +5,9 @@
   "type": "module",
   "scripts": {
     "download-keys": "tsx src/utils/download-proving-keys.ts",
-    "dev": "webpack --config webpack/webpack.dev.js --watch",
+    "dev": "webpack --watch --mode=development -d inline-source-map",
     "clean": "rm -rfv dist bin",
-    "build": "webpack --config webpack/webpack.prod.js",
+    "build": "webpack --mode=development -d inline-source-map",
```

Comment:
build should be `production` no?

Reply:
this is a script for developer convenience. if a develper is running `pnpm build` at the terminal, they are probably doing development, and want development features. in the unsual case they might want something else, the developer can run webpack directly and more explicitly.

in this PR, when CI makes a production build, it uses 

https://github.com/penumbra-zone/web/blob/43bf771a567a3705554fc13cd1ec9f038e9589e3/.github/workflows/extension-publish.yml#L41-L42

Reply:
okay, i changed this to just run `webpack` with no options, which defaults to `--mode=production` with no `devtool` (equivalent to before). the change to `extension-publish.yml` was reverted